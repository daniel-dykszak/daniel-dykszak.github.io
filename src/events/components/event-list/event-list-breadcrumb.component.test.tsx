import {render} from '@testing-library/react';
import {MemoryRouter} from "react-router-dom";
import {EventListBreadcrumb} from "./event-list-breadcrumb.component";

describe("EventListBreadcrumb component", () => {
    it("should match snapshot", () => {
        const component = render(<MemoryRouter>
            <EventListBreadcrumb/>
        </MemoryRouter>, {}).baseElement;
        expect(component).toMatchSnapshot();
    });
})