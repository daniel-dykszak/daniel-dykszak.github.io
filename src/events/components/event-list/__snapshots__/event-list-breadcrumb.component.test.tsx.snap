// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventListBreadcrumb component should match snapshot 1`] = `
<body>
  <div>
    <ul
      class="breadcrumb breadcrumb-container"
    >
      <li
        class="breadcrumb-item"
      >
        <a
          href="/"
          title="home"
        >
          Home
        </a>
      </li>
      <li
        class="breadcrumb-item breadcrumb-item__inactive"
      >
        Event list
      </li>
    </ul>
  </div>
</body>
`;
